[2021-08-16 15:47:21,657] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.update_entry 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:21,798] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_complex.update_entry 2021-08-16T15:43:47.414310+00:00 [queued]>
[2021-08-16 15:47:21,799] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:21,800] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-08-16 15:47:21,801] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-08-16 15:47:21,922] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): update_entry> on 2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:21,978] {standard_task_runner.py:52} INFO - Started process 386 to run task
[2021-08-16 15:47:22,075] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_complex', 'update_entry', '2021-08-16T15:43:47.414310+00:00', '--job-id', '35', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_complex.py', '--cfg-path', '/tmp/tmp_k7lphks', '--error-file', '/tmp/tmppwglmt6i']
[2021-08-16 15:47:22,079] {standard_task_runner.py:77} INFO - Job 35: Subtask update_entry
[2021-08-16 15:47:22,326] {logging_mixin.py:104} INFO - Running <TaskInstance: example_complex.update_entry 2021-08-16T15:43:47.414310+00:00 [running]> on host 8f195762143d
[2021-08-16 15:47:22,448] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_complex
AIRFLOW_CTX_TASK_ID=update_entry
AIRFLOW_CTX_EXECUTION_DATE=2021-08-16T15:43:47.414310+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-16T15:43:47.414310+00:00
[2021-08-16 15:47:22,461] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-08-16 15:47:22,466] {bash.py:158} INFO - Running command: echo update_entry
[2021-08-16 15:47:22,645] {bash.py:169} INFO - Output:
[2021-08-16 15:47:22,676] {bash.py:173} INFO - update_entry
[2021-08-16 15:47:22,677] {bash.py:177} INFO - Command exited with return code 0
[2021-08-16 15:47:22,732] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_complex, task_id=update_entry, execution_date=20210816T154347, start_date=20210816T154721, end_date=20210816T154722
[2021-08-16 15:47:22,842] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-16 15:47:22,902] {local_task_job.py:146} INFO - Task exited with return code 0
